sycl/test/check_device_code/native_cpu/target-features.cpp

```diff
@@ -0,0 +1,18 @@
+// REQUIRES: native_cpu
+// RUN: %clangxx --target=x86_64-unknown-linux-gnu -fsycl -fsycl-targets=native_cpu -mllvm -sycl-native-dump-device-ir %s 2>&1 | FileCheck %s --check-prefixes=CHECK,NOAVX
```

Comment:
Shouldn't we be testing LLVM IR emitted by clang in clang/test/CodeGenSYCL/ tests?

Reply:
Yeah, that's probably better, I'll move it, thanks.

Reply:
Done, the test is now `clang/test/CodeGenSYCL/native_cpu_target_features.cpp`.